infrastructure/infrastructure/pom.xml

```diff
@@ -11,37 +10,54 @@
 
     <dependencies>
         <dependency>
-            <groupId>com.google.inject</groupId>
-            <artifactId>guice</artifactId>
-            <version>5.0.1</version>
+            <groupId>dev.c0ps.franz</groupId>
```

Comment:
What is the `dev.c0ps.franz` pacakge? :)

Reply:
I have move the Kafka logic to a separate library. I called it "Franz" (homage to the writer Franz Kafka).